VolleyManagement.UI/spa/environments/environment.integrated.ts

```diff
@@ -0,0 +1,11 @@
+// The file contents for the current environment will overwrite these during build.
```

Comment:
"No need for this, at least now. Developer simply set environment.integrated to "true". And everything works. You should remember, we don't use specific flag with environment when running "ng serve"."

I did not get this one